integration-tests/Makefile

```diff
@@ -65,6 +65,11 @@ install_gotestfmt:
 build_test_image:
```

Comment:
when running the `make install_qa_tools` command, it fails on line 50 complaining file not found. The https link indeed gives 400.

> helm repo add chainlink-qa https://raw.githubusercontent.com/smartcontractkit/qa-charts/gh-pages/